forem/forem
# Update `GET /followers` API endpoint

## What type of PR is this? (check all applicable)


## Description

This PR makes 2 changes to the `GET /followers` API endpoint:

1. The `created_at` value of the follow is now included in the API response.
2. A sort order can be specified (before it was hard-coded to `created_at: :desc`).

Note: I used `sort` as the parameter name because we use the same name here: https://github.com/forem/forem/pull/13137/files

## Related Tickets & Documents

https://github.com/forem/internalEngineering/issues/404

## QA Instructions, Screenshots, Recordings

1. Verify that the `created_at` timestamp is part of the API response.
2. Verify that sorting works.

### UI accessibility concerns?

None, API only.

## Added tests?


## [Forem core team only] How will this change be communicated?
